src/util/getBreakpoints.js

```diff
@@ -1,38 +1,76 @@
 /**
+ * Returns the value found in obj at path. Delegates to obj.getIn if available.
+ * @param obj
+ * @param path
+ * @returns {*}
+ */
+function getIn(obj, path) {
+    if (obj.getIn) {
+        return obj.getIn(path)
+    }
+    return path.reduce((accum, next) => accum[next], obj)
+}
+
+/**
+ * Returns all keys of an object. Delegates to either obj.keys or Object.keys.
+ * @param obj
+ * @returns {*}
+ */
+
+function keys(obj) {
+    if (obj.keys) {
```

Comment:
Hm, i'm not sure I understand when this statement would evaluate to true. I looked through the immutable docs and I didn't see anything referencing this method - is there a problem with the usual`Object.keys`?

Reply:
`Object.keys` doesn't work as expected with Immutable `Map`s/`Record`s: see http://jsbin.com/vayazacami/1/edit?html,js,console
To work around this, they have `.keys()` method.

Reply:
ah perfect, makes total sense now. Thanks for the jsbin!